rpmconf/Upgrade/zmupgrade.pm

```diff
@@ -2047,6 +2047,14 @@ sub upgrade872GA {
     return 0;
 }
 
+sub upgrade886GA {
+    my ($startBuild, $targetVersion, $targetBuild) = (@_);
+    if (main::isInstalled("zimbra-proxy")) {
+        main::setLdapServerConfig($hn, 'zimbraReverseProxyStrictServerNameEnabled', "FALSE");
```

Comment:
Please can we have a comment explaining why the upgrade setting differs from the new install setting.

Reply:
Sure thing

Reply:
How does this look?